### ISSUE TITLE
[Bug]: Accessibility - DetailsList draggable columns have child divs without either correct role or aria-hidden=true specified

### ISSUE THREAD

**Issue by imatiach-msft** [microsoft/fluentui#31213]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: Windows 10 10.0.22631
    CPU: (8) x64 Intel(R) Core(TM) i7-1065G7 CPU @ 1.30GHz
    Memory: 11.38 GB / 31.60 GB
  Browsers:
    Edge: Chromium (123.0.2420.97)
    Internet Explorer: 11.0.22621.1
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codesandbox.io/

## Actual Behavior
For the class  .ms-DetailsHeader, there is accessibility error for Accessibility Insights:
Element has children which are not allowed: [role=presentation], div[aria-hidden]
This can be tracked to draggable columns, specifically to the line:
https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/DetailsList/DetailsHeader.base.tsx#L677

## Expected Behavior
This div should be set to area-hidden=true.

### Requested priority

Normal

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by Hotell** [comment]
can this be closed by https://github.com/orgs/microsoft/projects/395/views/11 ? @imatiach-msft

**Comment by imatiach-msft** [comment]
yes closing as this has been fixed, thank you for the reminder
